chirps_utility.py

```diff
@@ -0,0 +1,140 @@
+from chirps import *
+from main import *
+from csv_utility import *
+from user_utility import *
+from random_id_generator import *
+
+class ChirpsUtility:
+
+	def view_chirps(public_convos, private_convos, chirps_dict, current_user):
+		"""
+		Prints out all public and private chirps that the current user is a part of
+
+		Arg- chirps dictionary (from CSV file), currently selected user
+		"""
+		# counter and empty list to be used for convo select
+		# (display counter as choice number, append each convo to convo_id_list,
+		# convo int(counter - 1) to find convo in convo_id_list by index
+		counter = 1
+		convo_id_list = list()
+		# if a user is signed in
+		if current_user != None:
+			try:
+				print("\n ~ PRIVATE CHIRPS ~ ")
+				for convo_key, chirps_list in private_convos.items():
+					# chirp info that started convo
+					convo_start = chirps_list[0]
+					# if current user is the recipient of a private chirp
+					if chirps_dict[convo_start][1] == current_user.screen_name:
+						print("{0}. {1}: {2}".format(counter, chirps_dict[convo_start][0], chirps_dict[convo_start][2]))
+						convo_id_list.append(convo_key, "private")
+						counter += 1
+					# if current user sent a private chirp
```

Comment:
I like that you've split out display of private chirps based on whether the current user is the sender or recipient.